app/page.tsx

```diff
@@ -1,4 +1,8 @@
 import Button from "@/components/shared/Button";
+import Link from "next/link";
+import Image from "next/image";
+import Discord from "./Logos/Discord.svg";
+import LinkedIn from "./Logos/LinkedIn.svg";
```

Comment:
There is a folder for image assets. Check out `assets/images`. Can you move these social media svg files in there?

Reply:
Done @alexpereira.